ikarus/finiteelements/mechanics/kirchhoffloveshell.hh

```diff
@@ -234,6 +234,9 @@ protected:
     return kin;
   }
 
+  void updateStateImpl(const Requirement& /* par */,
```

Comment:
It should not be necessary to implement not needed functionality. 
This can be solved in the mixin class to only call that function if it exists.